rust/theoros/src/services/api/router.rs

```diff
@@ -17,6 +18,7 @@ pub fn api_router<T: OpenApiT>(state: AppState) -> Router<AppState> {
         .merge(SwaggerUi::new("/v1/docs").url("/v1/docs/openapi.json", open_api))
         .nest("/v1", calldata_routes(state.clone()))
         .nest("/v1", data_feeds_routes(state.clone()))
+        .nest("/v1", chains_routes(state.clone()))
```

Comment:
Let's reshape that, doesn't make sense to nest 3 things on the same route

Reply:
done